Examples/NoModules/Appendix A/ExA_06/ExA_06.cpp

```diff
@@ -0,0 +1,30 @@
+// Defining the same function twice
```

Comment:
Should you put a warning here that this file as-is will not compile?

Reply:
The reader should now if he's following the book... But sure, why not.